docs/sriov.md

```diff
@@ -1,4 +1,74 @@
+  SR-IOV resources before a pod or VMI is started. This component modifies host
+  resources to prepare them to be used inside a pod / VMI. Executed in root
+  network namespace context. Uses netlink and other commands to configure and
+  move SR-IOV VF / PF resources into pod namespaces.
+* [Multus](https://github.com/intel/multus-cni) is a meta-plugin. It will call
+  to SR-IOV CNI plugin when VMI is attached to an SR-IOV interface. It is
+  configured through `NetworkAttachmentDefinition` CRD objects. For SR-IOV, the
+  object annotations should refer to the resource name configured inside device
+  plugin `config.json` configuration file. This reference is used by KubeVirt
+  to automatically fill in `requests` and `limits` sections of `virt-launcher`
+  pods. (The same mechanism is used by other plugins, for example, `bridge`.)
+  If used alone, SR-IOV CNI plugin would need the PCI address of the device we
+  want to use inside the pod (and then, inside the VM). By using it in
+  combination with the device plugin and Multus, thanks to `resourceName`
+  annotation, the pod will receive a device from the pool allocated by the
+  device plugin, which allows to avoid manually passing the required PCI
+  address.
+* [OpenShift SR-IOV operator](https://github.com/openshift/sriov-network-operator)
+  configures kernel for SR-IOV, drains resources and reboots the kernel as
+  needed, deploys components listed above, and, based on values in policy CRD
+  resources managed by admin, configures the above listed components and the
+  host.
+* KubeVirt, based on values of environment variables set by SR-IOV device
+  plugin, configures libvirt domain for SR-IOV attached VMIs to use the right
+  PCI IDs. Also, fills in `requests` and `limits` sections of `virt-launcher`
+  pod spec as per attached NetworkAttachmentDefinition CRD.
+
+# Configuration
+
+> NOTE: steps to configure SR-IOV on a host are only applicable if you don't
+> use OpenShift SR-IOV operator to deploy SR-IOV components. Despite the name,
+> the operator supports installation on a plain Kubernetes cluster. The
+> operator handles most of the steps discussed below, including configuring
+> kernel parameters. The only step that cannot be handled by the operator is
+> BIOS setup, which should be handled by other means (either manually or
+> through some kind of IPMI driven automation).
+>
+> Documentation on how to use the operator
+> [is located](https://github.com/openshift/sriov-network-operator/blob/master/README.md)
+> [elsewhere](https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md).
+> Just remember to set `SriovNetworkNodePolicy` to use `deviceType: vfio-pci`
+> when configuring the operator.
+>
+> For local development purposes, one should be able to (re)use the same
+> [kind](https://kind.sigs.k8s.io/) based provider that the official upstream
+> SR-IOV CI relies on, for example:
+>
+> $ export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.14.2
```

Comment:
following two lines are rendered on the same one, you may need to make them into two paragraphs